.circleci/scripts/maps.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eux -o pipefail
```

Comment:
What is this line for?

Reply:
Just copy from other bash files

Reply:
But this file only contain one function, right?

Reply:
I see. It's useless, I'll remove it